app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.name = name
```

Comment:
The attribute `self.name` is being assigned twice. The second assignment on line 7 is redundant and should be removed.